point-to-point/p2pchannel/p2p.go

```diff
@@ -1,26 +1,27 @@
 package p2pchannel
 
 import (
-	"github.com/streadway/amqp"
 	"log"
+
+	"github.com/streadway/amqp"
 )
 
-type P2p struct {
+type P2P struct {
 	xchng      string
 	queueName  string
 	routingKey string
 	conn       *amqp.Connection
 }
 
-//We just use a contant here for convenience, in reality you configure this
+//We just use a constant here for convenience, in reality you'd configure this
 const exchange = "practical-messaging-go"
 
-func NewChannel(qName string) *P2p {
-
-	channel := new(P2p)
-	channel.xchng = exchange
-	channel.queueName = qName
-	channel.routingKey = qName
+func NewChannel(qName string) *P2P {
+	channel := &P2P{
```

Comment:
General guidelines prefer initialisation via struct literal as opposed to the `new()` function.

Reply:
Also, acronyms are all uppercase (ie: Http is HTTP, etc)

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms